docs/resources/mlflow_models.md

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "MLFlow"
+---
+# databricks_mlflow_model Resource
+
+This resource allows you to create MLFlow models in Databricks.
+
+## Example Usage
+
+```hcl
+resource "databricks_mlflow_model" "test" {
+  name = "My MLFlow Model"
+
+  description = "My MLFlow model description"
+
+  tags {
+    key   = "key1"
+    value = "value1"
+  }
+  tags {
+    key   = "key2"
+    value = "value2"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Name of MLFLow model.
```

Comment:
There are several case problems throughout the documents: `MLFlow` and `MLFLow`.